src/__tests__/parser.ts

```diff
@@ -430,10 +430,6 @@ describe('parser', () => {
       check('StatelessWithDefaultProps', expectation);
     });
 
-    it('should parse referenced props', () => {
```

Comment:
Deleted these 2 tests (`should parse referenced props` and `supports spread props`) because this use cases are not actually supported and default props are not properly generated. These tests were not failing on master because of incorrect check inside test code itself: see https://github.com/styleguidist/react-docgen-typescript/pull/200/files#diff-a063097dc09ae77820847611faefb99b

Reply:
Thanks for explanation.